OpenRA.Mods.Common/Projectiles/Missile.cs

```diff
@@ -224,6 +224,7 @@ public Missile(MissileInfo info, ProjectileArgs args)
 			gravity = new WVec(0, 0, -info.Gravity);
 			targetPosition = args.PassiveTarget;
 			rangeLimit = info.RangeLimit != WDist.Zero ? info.RangeLimit : args.Weapon.Range;
+			rangeLimit = new WDist(Util.ApplyPercentageModifiers(rangeLimit.Length, args.RangeModifiers));
```

Comment:
Assigning `rangeLimit` twice looks kind of odd, what about
```suggestion
			var limit = info.RangeLimit != WDist.Zero ? info.RangeLimit : args.Weapon.Range;
			rangeLimit = new WDist(Util.ApplyPercentageModifiers(limit.Length, args.RangeModifiers));
```